html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../../html-relative-links/index.html">Home Page</a></li>
```

Comment:
We are getting there! Again it could cause a problem if we change our root folder name. So think as if you are starting from `about-me` folder in this case — how do we get from `about-me` to the directory where it has `index.html` without going through the root directory `html-relative-links`?